docs/diginstroom/sip/1.0/profiles/newspaper.md

```diff
@@ -0,0 +1,346 @@
+---
+layout:       default
+title:        Newspaper
+parent:       Profiles
+nav_order:    2
+nav_exclude:  false
+---
+# Profile: Newspaper 
+
+The newspaper profile supports the ingest of digitised newspaper content.
+It is an extension of the generic SIP specification and it shows how to deal with multiple media files (in formats such as TIFF, ALTO XML and PDF) and the relationships between them and their metadata.
+It also allows to use the [MODS metadata schema](https://www.loc.gov/standards/mods/) for descriptive metadata, which is the default for describing newspaper content.
+
+**Permalink:** <https://data.hetarchief.be/id/sip/1.0/newspaper>
+
+**Example Directory structure:**
+
+```plaintext
+root_directory
+│──manifest-md5.txt
+│──bagit.txt
+│
+└──data
+    │──mets.xml
+    │──metadata
+    |   |──descriptive
+    |   |  └──dc.xml
+    |   |  └──mods.xml
+    |   └──preservation
+    |       └──premis.xml
+    │
+    └──representations
+        │──representation_1
+        │    │──mets.xml
+        │    │──data
+        │    |   |──file_1.tiff
+        │    │   └──...
+        │    │
+        │    └──metadata
+        │        └──preservation
+        │            └──premis.xml
+        │
+        │──representation_2
+        │    │──mets.xml
+        │    │──data
+        │    |   |──file_1.xml
+        │    │   └──...
+        │    │
+        │    └──metadata
+        │       └──preservation
+        │          └──premis.xml
+        │
+        └──representation_3
+            │──mets.xml
+            │──data
+            |  └── file_1.pdf
+            │
+            └──metadata
+               └──preservation
+                  └── premis.xml
+```
+
+## Requirements
+
+### General
+
+- A newspaper SIP MUST contain exactly one newspaper edition.
+- The newspaper edition MUST be digitised per page, i.e. each TIFF and/or ALTO XML file contained in their respective representation directories MUST represent exactly one page.
+  - <a id="pdf"></a>An exception to this requirement MAY be made with regards to the PDF file: if a PDF file is present, it SHOULD contain the contents of the entire newspaper edition (i.e. all pages are present in one single PDF file).
+- There MUST be exactly one IE present in the SIP, i.e. the newspaper edition.
+- Preservation metadata in the SIP MUST be limited to the PREMIS metadata schema.
+- There MUST be preservation metadata at the package level in the `preservation/premis.xml` file.
+- There MUST be preservation metadata at the representation level in the respective `preservation/premis.xml` files.
+- There MUST NOT be any descriptive metadata at the representation level.
```

Comment:
I suppose you could include metadata at the representation level when you digitise per article instead of per page (cf. EHC/SA Mechelen). In that case each file represents an article instead of a page and it would be possible in such a case to include e.g. the author of an article. On the other hand, in that case you would probably have to add separate IEs for each article and then such metadata would be included at the package level and not at the representation level.

Perhaps something about the OCR process that deals specifically with the alto XML files? But such information would most likely be included in the preservation metadata rather than the descriptive metadata...

Do you have anything in mind re. descriptive metadata that would occur at the representation level?

Reply:
A title and a description, for instance, to explain that this is a high-resolution photoregistration of the edition.